src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -4,21 +4,43 @@ import { IPoint } from '@uifabric/utilities';
 import { ActionButton } from 'office-ui-fabric-react/lib/Button';
 import { ContextualMenu, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
 import * as React from 'react';
+import { IssueDetailsTextGenerator } from '../../../background/issue-details-text-generator';
+import { DetailsViewActionMessageCreator } from '../../../DetailsView/actions/details-view-action-message-creator';
+import { createDefaultLogger } from '../../logging/default-logger';
+import { Logger } from '../../logging/logger';
+import { NavigatorUtils } from '../../navigator-utils';
+import { WindowUtils } from '../../window-utils';
+import { Toast } from '../toast';
 import { kebabMenuButton } from './card-footer.scss';
 
+export type CardKebabMenuButtonDeps = {
+    windowUtils: WindowUtils;
+    issueDetailsTextGenerator: IssueDetailsTextGenerator;
+    detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
+    navigatorUtils: NavigatorUtils;
+};
+
 export interface CardKebabMenuButtonState {
     isContextMenuVisible: boolean;
     target?: HTMLElement | string | MouseEvent | IPoint | null;
+    showingCopyToast: boolean;
+    showNeedsSettingsContent: boolean;
```

Comment:
`showNeedsSettingsContent` property on the state is only being set on the constructor but never actually used. Should be removed until we actually need it.